### PR TITLE
Refactor/functionality

### PR DESCRIPTION
**Pull request by KostiaLevchenko**

### CODE REVIEW

File: src/lib/includes/kfs_common.h [hunk 1]
```diff
@@ -8,111 +8,19 @@
 #include <stdio.h>
 #include <zconf.h>
 #include <dirent.h>
+#include <assert.h>
```
Comment (kopachevsky) [inline src/lib/includes/kfs_common.h:11]: why it needed?

File: src/lib/kfs_flush.c [hunk 2]
```diff
@@ -1,6 +1,8 @@
 #include "kfs_flush.h"
 
-int kfs_flush(const char* path, struct fuse_file_info *fi) {
+int kfs_flush(const char* original_path, struct fuse_file_info *fi) {
+    char *path = local_disk_cache_path(original_path);
+    (void) path;
     int res = close(fi->fh);
     if (res == -1)
         return -errno;
```
Comment (kopachevsky) [inline src/lib/kfs_flush.c:8]: where is free(path)?

File: tests/kfs_access_test.c [hunk 3]
```diff
@@ -69,7 +69,7 @@ START_TEST(kfs_access_rwx) {
     char dir_path[strlen(LOCAL_DISC_CACHE_PATH) + strlen("rwx.txt") + 1];
     strcpy(dir_path, LOCAL_DISC_CACHE_PATH);
     char *path = strcat(dir_path, "rwx.txt");
-    struct fuse_file_info create = {O_CREAT};
+    struct fuse_file_info create = {O_CREAT,0,0,1,1,0,1,1,27,0,0};
```
Comment (kopachevsky) [inline tests/kfs_access_test.c]: replace with function eventually

**Review by kopachevsky — APPROVED** [review]

File: src/lib/includes/kfs_common.h (hunk 1 above)
Comment (KostiaLevchenko) [reply src/lib/includes/kfs_common.h:11]: for str_concat (kfs_common line 4 and 7)

File: src/lib/kfs_flush.c (hunk 2 above)
Comment (kopachevsky) [reply src/lib/kfs_flush.c:8]: and?

**Pull request merged**